features/profile/view/leaveReference/ReferenceForm.tsx

```diff
@@ -85,7 +86,16 @@ export default function ReferenceForm({
     }));
   };
 
-  return step === "appropriate" ? (
+  const isSkippedStep =
+    referenceData.wasAppropriate === "" && step !== "appropriate";
```

Comment:
There are a few steps - does this just check if the first was skipped?

Reply:
yup. right now `missing the first step <=> user has to start from the beginning`